docs/symbiflow_investigation.md

```diff
@@ -0,0 +1,100 @@
+## Getting acquainted with the open-source "Symbiflow" package for synthesizing, implementing and generating the FPGA configuration file
+
+Here is a link to the main Github repository for the **Symbiflow** software package:
+
+[Github Symbiflow repository](https://github.com/SymbiFlow)
+
+First, I installed a modified version of the **Yosys** package (modifications for Symbiflow) from this report:
+
+[Github Yosys Symbiflow repository](https://github.com/SymbiFlow/yosys)
+
+Installation carried out according to the instructions on the main page of this repository. Yosys is only for FPGA synthesis phase. A list of Verilog primitive connections is created in ** JSON ** format.
+
+The next tool is for the implementation phase (map processes, placement and routing). For this phase you need a "user constaraints" file in ** Pcf ** format for the selected FPGA.
```

Comment:
@mgabryelski1 Please wrap lines at 80 characters. Probably you do not use pre-commit - please install it for this repository: https://github.com/3mdeb/openpower-coreboot-docs/blob/main/devnotes/tpm.md#ftpm-in-sbe